spec/requests/searches_spec.rb

```diff
@@ -25,7 +28,9 @@
     context "when the request is from a mobile device" do
       before do
         iphone_user_agent = "Mozilla/5.0 (iPhone; U; CPU like Mac OS X; en) AppleWebKit/420+ (KHTML, like Gecko) Version/3.0 Mobile/1A543a Safari/419.3"
-        get "/search/docs", {:query => "pdf", :affiliate => affiliate.name}, { "HTTP_USER_AGENT" => iphone_user_agent }
+        get "/search/docs", params: { query: "pdf",
+                                      affiliate: affiliate.name },
+                            headers:  { "HTTP_USER_AGENT" => iphone_user_agent }
       end
```

Comment:
Can this be written headers: {  'HTTP_USER_AGENT': iphone_user_agent }

Reply:
I'll switch to single quotes, but we should keep these as string keys, as that's the standard format for headers:
https://api.rubyonrails.org/classes/ActionDispatch/Http/Headers.html